Gems/LmbrCentral/Code/lmbrcentral_headers_files.cmake

```diff
@@ -0,0 +1,63 @@
+    include/LmbrCentral/Ai/NavigationComponentBus.h
+    include/LmbrCentral/Ai/NavigationAreaBus.h
+    include/LmbrCentral/Ai/NavigationSystemBus.h
+    include/LmbrCentral/Ai/NavigationSeedBus.h
+    include/LmbrCentral/Animation/AttachmentComponentBus.h
+    include/LmbrCentral/Animation/SkeletalHierarchyRequestBus.h
+    include/LmbrCentral/Audio/AudioEnvironmentComponentBus.h
+    include/LmbrCentral/Audio/AudioListenerComponentBus.h
+    include/LmbrCentral/Audio/AudioMultiPositionComponentBus.h
+    include/LmbrCentral/Audio/AudioPreloadComponentBus.h
+    include/LmbrCentral/Audio/AudioProxyComponentBus.h
+    include/LmbrCentral/Audio/AudioRtpcComponentBus.h
+    include/LmbrCentral/Audio/AudioSwitchComponentBus.h
+    include/LmbrCentral/Audio/AudioSystemComponentBus.h
+    include/LmbrCentral/Audio/AudioTriggerComponentBus.h
+    include/LmbrCentral/Bundling/BundlingSystemComponentBus.h
+    include/LmbrCentral/Geometry/GeometrySystemComponentBus.h
+    include/LmbrCentral/Dependency/DependencyMonitor.h
+    include/LmbrCentral/Dependency/DependencyMonitor.inl
+    include/LmbrCentral/Dependency/DependencyNotificationBus.h
+    include/LmbrCentral/Physics/WindVolumeRequestBus.h
+    include/LmbrCentral/Physics/ForceVolumeRequestBus.h
+    include/LmbrCentral/Physics/WaterNotificationBus.h
+    include/LmbrCentral/Rendering/DecalComponentBus.h
+    include/LmbrCentral/Rendering/LightComponentBus.h
+    include/LmbrCentral/Rendering/MaterialAsset.h
+    include/LmbrCentral/Rendering/MaterialHandle.h
+    include/LmbrCentral/Rendering/MeshAsset.h
+    include/LmbrCentral/Rendering/MeshModificationBus.h
+    include/LmbrCentral/Rendering/RenderNodeBus.h
+    include/LmbrCentral/Rendering/GiRegistrationBus.h
+    include/LmbrCentral/Rendering/RenderBoundsBus.h
+    include/LmbrCentral/Scripting/EditorTagComponentBus.h
+    include/LmbrCentral/Scripting/GameplayNotificationBus.h
+    include/LmbrCentral/Scripting/SimpleStateComponentBus.h
+    include/LmbrCentral/Scripting/SpawnerComponentBus.h
+    include/LmbrCentral/Scripting/RandomTimedSpawnerComponentBus.h
+    include/LmbrCentral/Scripting/TagComponentBus.h
+    include/LmbrCentral/Shape/EditorShapeComponentBus.h
+    include/LmbrCentral/Shape/ShapeComponentBus.h
+    include/LmbrCentral/Shape/SphereShapeComponentBus.h
+    include/LmbrCentral/Shape/BoxShapeComponentBus.h
+    include/LmbrCentral/Shape/CylinderShapeComponentBus.h
+    include/LmbrCentral/Shape/CapsuleShapeComponentBus.h
+    include/LmbrCentral/Shape/DiskShapeComponentBus.h
+    include/LmbrCentral/Shape/CompoundShapeComponentBus.h
+    include/LmbrCentral/Shape/QuadShapeComponentBus.h
+    include/LmbrCentral/Shape/SplineComponentBus.h
+    include/LmbrCentral/Shape/PolygonPrismShapeComponentBus.h
+    include/LmbrCentral/Shape/TubeShapeComponentBus.h
+    include/LmbrCentral/Shape/SplineAttribute.h
+    include/LmbrCentral/Shape/SplineAttribute.inl
+    include/LmbrCentral/Terrain/TerrainSystemRequestBus.h
+)
```

Comment:
can we add a newline at the end of this file for linux?

Reply:
Done